crypto/store/store_init.c

```diff
@@ -14,7 +14,7 @@
 static CRYPTO_ONCE store_init = CRYPTO_ONCE_STATIC_INIT;
 DEFINE_RUN_ONCE_STATIC(do_store_init)
 {
-    return OPENSSL_init_crypto(0, NULL);
+    return 1;
```

Comment:
Actually.....can we remove this run once altogether now?

Reply:
I thought you'll ask. :grin: